Code/Framework/AzCore/AzCore/Debug/Trace.cpp

```diff
@@ -558,6 +558,7 @@ namespace AZ::Debug
         fwrite(windowView.data(), 1, windowView.size(), stdout);
         fwrite(windowMessageSeparator.data(), 1, windowMessageSeparator.size(), stdout);
         fwrite(messageView.data(), 1, messageView.size(), stdout);
+        fwrite("\n\r", 1, 2, stdout);
```

Comment:
First this is windows specific: The carriage return isn't actually needed when writing a newline to stdout
Second the carriage reutrn should be written before the newline.

Most importantly, it is not the responsibility of this function to add a newline.
It looks like what you want to do is to add support to invoke `fflush(stdout)` parameter to the RawOutput function.

Reply:
fflush is nice since that does help the editor get the new stdout instantly (not just every few seconds when a flush event is triggered). Although, without the server printing newline, I'm not sure of a way for the editor (listening to the server's out) to differentiate between each individual log. Right now the editor will only pump once a tick, and that can result in a couple logs lines being read at once.
For now I've reverted Trace.cpp entirely. I'll look into hand-adding newlines to any logs the editor-server really cares about. The biggest issue might be that (as far as I can tell so far) script canvas debug prints dont support newlines, so I might need to add that ability.